### ISSUE TITLE
[v10] Game Settings are not displayed if a module is breaking the settings.

### ISSUE THREAD

**Issue by vt-tom** [sdenec/tidy-ui_game-settings#85]
If i activate Tidy UI the Game Settings are Shown empty (testet in dnd5e). This is in combination with other modules that are not showing the settings properly. For excample - SmallTime:

Smalltime Settings: 
![grafik](https://user-images.githubusercontent.com/80099175/188586994-0fe0805d-bba0-446c-85de-587e4071137a.png)

Tidy UI active:
![grafik](https://user-images.githubusercontent.com/80099175/188587746-b18c6a41-ccb9-4f48-9736-f2d5151eeb00.png)

If i drag the window bigger:
![grafik](https://user-images.githubusercontent.com/80099175/188587929-7a0f495c-4062-414c-b0d7-b52b04489035.png)

**Comment by sdenec** [comment]
check if you are on 0.1.5
Tidy for v10 doesn't touch the setting window. Just modules.
The behaviour you are describing is the same with or without tidy enabled

**Comment by vt-tom** [comment]
I am on v10 and tidy 0.1.5 and if i activate tidy it is shown as in the 2 picture at the bottom of my initial post. If i deactivate tidy it is as in the top picture. So it makes a little difference using tidy or not. 
I tested another thing. I deactivated smalltime and now im getting this result with tidy:
![grafik](https://user-images.githubusercontent.com/80099175/188619545-47b282a4-4889-4405-91ba-d3cee50c1a4b.png)
